run/src/index.ts

```diff
@@ -229,6 +231,63 @@ export function setup<So extends Sources, Si extends FantasySinks<Si>>(
   return {sinks, sources, run: _run};
 }
 
+/**
+ * A partially-applied variant of setup() which accepts only the drivers, and
+ * allows many `main` functions to execute and reuse this same set of drivers.
+ *
+ * Takes an object with driver functions as input, and outputs an object which
+ * contains the generated sources (from those drivers) and a `run` function
+ * (which in turn expects sinks as argument). This `run` function can be called
+ * multiple times with different arguments, and it will reuse the drivers that
+ * were passed to `setupReusable`.
+ *
+ * **Example:**
+ * ```js
+ * import {setupReusable} from '@cycle/run';
+ * const {sources, run} = setupReusable(drivers);
+ * const sinks = main({...sources, foo: fooDriver});
```

Comment:
I would assume you can't pass in a driver directly as sources, do you?

Reply:
Good point!